flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceSql.java

```diff
@@ -162,4 +162,21 @@ public void testWatermarkOptionsDescending() throws Exception {
         expected,
         SCHEMA_TS);
   }
+
+  /** Test create table using LIKE */
+  @Test
+  public void testFlinkTableUsingLIKE() throws Exception {
```

Comment:
nit: rename to `testCreateTableLikeDiffCatalog`

Reply:
renamed to testReadFlinkDynamicTable